interface/forms/fee_sheet/review/js/fee_sheet_core.js

```diff
@@ -136,7 +138,7 @@ function justify_start(evt) {
         }
     }).done(function (data) {
         // now init justify
-        let parent = jqElem.parent()
+        let parent = jqElem.parent();
```

Comment:
guess we should've noticed this in the disappearing modifier :1st_place_medal:

Reply:
yep, I have a bad habit of missing the small stuff too.  I rely to much on cs fixing for me but I don't have turned on for JS.

Reply:
neither does the project obviously, if managed to update codesniff that would be quite the undertaking :plate_with_cutlery:

Reply:
i've been tempted to format them if I touch one but then it gets confusing for reviewer. Best to do all at once one day like was done for php scripts in the beginning...

Reply:
or this one https://github.com/openemr/openemr/pull/2527/commits/118a2de66a0d8e732455e0dcebf28e845661ac51 :)